ailab/ailab/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for ailab project.
+
+Generated by 'django-admin startproject' using Django 1.11.18.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'eo+li9#pw2im0xeq3@bv0h0ir^o36w&do%&3nm!!+!bxz5q9my'
```

Comment:
I don't think you can expose this. It is supposed to be a secret.

Reply:
Will implement a mechanism to generate a random secret key the first time the project is started.